### PR TITLE
8314 task: move files to CR

### PR DESCRIPTION
**Pull request by skibinska**
Relates https://github.com/wellcometrust/corporate/issues/8314

### This PR

- removes  `.scaffolding.scss`
- removes `sticky-fade` mixin (I checked it was used ones in `.scaffolding.scss`)
- adds `_hacks.sccs` file
- there are now in CR

### CODE REVIEW

File: src/assets/styles/50-utilities/_scaffolding.scss [hunk 1]
```diff
@@ -40,3 +40,9 @@
     @include grid-column(1, 13);
   }
 }
+
+.u-scaffold-grid-item--extended {
+  @include mq(md) {
+    @include grid-column(4, 13);
+  }
+}
```
Comment (chriddell) [inline src/assets/styles/50-utilities/_scaffolding.scss]: I'm very surprised to find these styles in corporate-components. I feel they should be declared in corporate-react as the component they apply to only exists in corporate-react.

What do you think @creido-welly?

Comment (creido-welly) [reply src/assets/styles/50-utilities/_scaffolding.scss]: Yes it makes sense to move them

**Review by chriddell — APPROVED** [review]
The best kind of PR = one where there only deletions 💫

**Review by chriddell — APPROVED** [review]
👍

**Pull request merged**